data/datacatalogue/CatalogueOntologies/Areas of information ds.csv

```diff
@@ -1,194 +1,240 @@
-name,label,definition,parent
```

Comment:
How did you check that this one didn't break stuff?

Reply:
Break stuff? I synchronized the existing databases to these terms, which was almost one to one mapping.